frontend/src/components/selects/CategorySelect.js

```diff
@@ -1,35 +1,81 @@
 import React from 'react';
-import { Button, MenuItem } from '@blueprintjs/core';
+import { Menu, Button, MenuItem } from '@blueprintjs/core';
 import { Select } from '@blueprintjs/select';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
+import deselectButton from '../../resources/deselect.png';
 
 const SelectContainer = styled.div``;
 
-const categories = ['Medical Equipment', 'World Vision', 'Bio Med', 'Kendall'];
+const ButtonWithBorder = styled.button`
```

Comment:
Replace this with the BlueprintJS [Button component](https://blueprintjs.com/docs/#core/components/button) and make it an Icon button.